core/src/commonMain/kotlin/State.kt

```diff
@@ -2,7 +2,11 @@ package com.juul.kable
 
 public sealed class State {
 
-    public object Connecting : State()
+    public sealed class Connecting : State() {
+        public object Bluetooth : Connecting()
+        public object Services : Connecting()
+        public object Observers : Connecting()
```

Comment:
I'm torn if `Observes` or `Observations` would be better?

We are indicating that we're wiring up the observations for flows created via calls to `observe(..)` function. 🤷

Reply:
Observations seems best to me? Seems like we're waiting until observations are set up, not the observers

Reply:
Went with observes in this scenario.